Document/0x05c-Reverse-Engineering-and-Tampering.md

```diff
@@ -573,6 +573,50 @@ main[1] cont
 
 This is the plaintext string you're looking for!
 
+The `java.lang.String.equals` breakpoint might be too common, so it can be reached multiple times, even before you are able to enter arbitrary text in the edit field, and tap the "verify" button. In this case, it is recommended to clear the `java.lang.String.equals` breakpoint temporarily, and set a new breakpoint in the interesting method that performs the string comparisson. Remember, from the "static analysis" section, the method was `sg.vantagepoint.uncrackable1.a.a`.
```

Comment:
> The `java.lang.String.equals` breakpoint might be too common, so it can be reached multiple times, even before you are able to enter arbitrary text in the edit field, and tap the "verify" button.

I understand what you mean here, but I think these are too many sentences strung together with comma's.

> comparisson

comparison

Reply:
I agree @Sjord. I have separated it in multiple sentences.